hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,32 +1,30 @@
-# USB mass storage device boot
+# USB mass storage boot
 
-**USB mass storage boot is available on Raspberry Pi 3B, 3B+, 3A+, and 2B v1.2 only.**
+**USB mass storage boot is available on Raspberry Pi 2B v1.2, 3A, 3A+, 3B, 3B+ only. Support for USB mass storage boot will be added to the Pi 4B in a future software update.**
 
-This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or USB hard disk. Be warned that this feature is experimental and does not work with all USB mass storage devices. See [this blog post](https://www.raspberrypi.org/blog/pi-3-booting-part-i-usb-mass-storage-boot/) from Gordon Hollingworth for an explanation of why some USB mass storage devices don't work, as well as for some background information.
+This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or USB hard disk. Note that this feature does not work with all USB mass storage devices.
 
-## Program USB boot mode
+## Program USB host boot mode (Pi 3A, 3A+ and 3B only)
 
-The Raspberry Pi 3A+ and 3B+ are able to boot from USB without any changes, but the Raspberry Pi 3A and 3B require the USB boot bit to be set in the OTP (one-time programmable) memory. If you are using a Raspberry Pi 3A+ or 3B+, please go to the next section.
-
-To enable the USB boot bit, the Raspberry Pi 3 needs to be booted from an SD card with a `config` option to enable USB boot mode. 
+To enable the USB host boot mode bit, the Pi needs to be booted from an SD card with a special option to enable USB host boot mode. 
 
 Once this bit has been set, the SD card is no longer required. **Note that any change you make to the OTP is permanent and cannot be undone.**
 
-You can use any SD card running Raspbian or Raspbian Lite to program the OTP bit. If you don't have such an SD card then you can install Raspbian or Raspbian Lite in the normal way - see [installing images](../../../installation/installing-images/README.md).
-
-First, prepare the `/boot` directory with up to date boot files (this step is not required if you're using the 2017-04-10 release of Raspbian/Raspbian Lite or a later one):
+**On the Raspberry Pi 3A+, setting the OTP bit to enable USB host boot mode will permanently prevent that Pi from booting in USB device mode.**
```

Comment:
Is this specific to the 3A+, or is the same true for the 3A too?

Reply:
I'm pretty sure it is just the 3A+. I'm guessing the OTGID pin on the 3A is done the same way as on the original A. There was a forum post from Gordon I looked at when I made that change - I will need to see if I can find it.

Reply:
Doh, brain-fart on my part! (I blame it on being Sunday)
There's no such thing as the "Raspberry Pi 3A", I got confused by this document mentioning both "Pi 3A" and "Pi 3A+" :upside_down_face: :rofl:

Reply:
Oops - my mistake. Why did I think there was a Pi 3A? 🤔